app/code/Magento/Review/Observer/CatalogBlockProductListCollectionInitializeObserver.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Review\Observer;
+
+use Magento\Framework\Event\ObserverInterface;
+
+class CatalogBlockProductListCollectionInitializeObserver implements ObserverInterface
```

Comment:
Name of observer should reflect what it actually does. To me it looks like it appends some summary.

Reply:
Changed